src/main/java/com/dabsquared/gitlabjenkins/trigger/handler/note/NoteHookTriggerHandlerImpl.java

```diff
@@ -67,8 +73,8 @@ protected CauseData retrieveCauseData(NoteHook hook) {
                 .withTargetProjectId(hook.getMergeRequest().getTargetProjectId())
                 .withBranch(hook.getMergeRequest().getSourceBranch())
                 .withSourceBranch(hook.getMergeRequest().getSourceBranch())
-                .withUserName(hook.getMergeRequest().getLastCommit().getAuthor().getName())
-                .withUserEmail(hook.getMergeRequest().getLastCommit().getAuthor().getEmail())
+                .withUserName(hook.getUser().getUsername())
```

Comment:
This is not backward compatible. People that are relying on this being the user name and email of committer will see bugs. 

Might be useful to create new field that deals with the hook user.